packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -168,7 +181,51 @@ const GROUPED_ACTIONS = [
 
   const getNPendingTasks = getPendingTasks.count()
 
-  const getPendingTasksByPool = getPendingTasks.sort().groupBy('$pool')
+  const predicate = createSelector(
+    createSelector(
+      getPendingTasks,
+      createCollectionWrapper(_ => keyBy(_, 'id'))
+    ),
+    tasksById => obj =>
+      !isEmpty(pick(tasksById, Object.keys(obj.current_operations)))
+  )
+
+  const getLinkedObjectsByTask = createSelector(
+    createGetObjectsOfType('pool').filter(predicate),
+    createGetObjectsOfType('host').filter(predicate),
+    createGetObjectsOfType('SR').filter(predicate),
+    createGetObjectsOfType('VDI').filter(predicate),
+    createGetObjectsOfType('VM').filter(predicate),
+    createGetObjectsOfType('network').filter(predicate),
+    (pools, hosts, srs, vdis, vms, networks) => {
+      const linkedObjectsByTask = {}
+      forEach(
+        { ...pools, ...hosts, ...srs, ...vdis, ...vms, ...networks },
```

Comment:
Spreading then reaching a big collection will cost perf. Simply reach each type of object with a mutualized CB.

Example: 
```js
forOwn(pools, cb)
forOwn(hosts, cb)
```